app/Services/InviteService.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
@Babita3331 Seems you have reused the code from the time-tracker. Fine, but there are tickets created in their board to not have static methods. The same issue will be raised here